README.adoc

```diff
@@ -86,83 +86,45 @@ include::{common-includes}/gitclone.adoc[]
 == Creating the Dockerfile
 
 A Dockerfile is a collection of instructions to build a Docker image that you can then
-run as a container. Every Dockerfile begins
-with a parent or base image on top of which these instructions are run. For example, you can start
-your image from scratch and execute commands that download Java, or you can start from
-an image that already contains a Java installation.
-
-Use the `openliberty/open-liberty:javaee7` image that contains the Open Liberty
-runtime as the parent image.
-
-Before you write your Dockerfile, familiarize yourself with the types of instructions that are compatible
-with Dockerfiles. Use the following instructions to build a Dockerfile for this guide:
-
-[cols="70, 100", options="header"]
-|===
-| *Command* | *Description*
-| FROM <image> | This command initializes a new build stage and indicates the parent image from which your image
-is built, or you can use `FROM scratch` to indicate that your image is a base image.
-| RUN <commands> | This command executes the commands in a new layer on top of the current image.
-| CMD ["executable", "param1", "param2"] | This command defines the command to be executed when the image runs.
-| ENTRYPOINT ["executable", "param1", "param2"] | This command works similarly to the `CMD` command, but unlike
-`CMD`, only the `--entrypoint` flag can override `ENTRYPOINT`.
-|===
-
-For a more extensive and detailed list, see the official https://docs.docker.com/engine/reference/builder[Docker documentation].
-
-=== Writing your Dockerfile
-
-To begin, specify `openliberty/open-liberty:javaee7` as the parent image. This image contains the Open Liberty runtime.
+run as a container. Every Dockerfile begins with a parent or base image on top of which these instructions
+are executed. For example, you can start your image from scratch and execute commands that download Java,
```

Comment:
This part of the sentence is passive voice: "on top of which these instructions are executed." The instructions are being acted upon.

What about something like..."Every Dockerfile begins with a parent or base image. Execute these instructions on top of the parent or base image."

Reply:
Its meant to be like that. The "instructions" are from the previous sentences. I basically tried to say that a Dockerfile is a collection of various instructions, and these instructions execute on top of the parent image.

Reply:
We try to avoid self-referential language, such as saying "this guide" (but understand if we use it sometimes). Would it be possible in this context to omit the mention of "these instructions"?

Maybe..."Every Dockerfile begins with a parent or base image. You can start your image from scratch..."

Reply:
ah ok. Then maybe we could rewrite the whole paragraph into something a bit different? This is what I came up with:

"A Dockerfile is a collection of instructions for building a Docker image that can then be run as a container. Every Dockerfile begins with a parent or base image on top of which various commands are run. For example, you can start your image from scratch and execute commands that download and install Java, or you can start from an image that already contains a Java installation."

Reply:
Yes, I like the revised paragraph that you have here. :)